jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc

```diff
@@ -79,50 +82,56 @@ The instructions below guide you through cloning the repository containing the K
 
 [source,shell]
 ----
-############ HELM Commands ################
-Output file is here - <path>/kots-exporter/output/helm-values.yaml
+############ Upgrade Commands ################
+Before upgrading to 4.0, follow below steps:
 
-To upgrade Postgres Chart(v11.6.0), follow below steps -
+Your helm values file with your installation config is here:
+- <path>/kots-exporter/output/helm-values.yaml
 -------------------------------------------------------------------------
-export POSTGRESQL_PASSWORD=$(kubectl get secret --namespace <namespace> postgresql -o jsonpath="{.data.postgres-password}" | base64 --decode)
+## Progres Chart Upgrade Preparation
+Upgrading to server CircleCI server 4.0 includes upgrading the Posgres chart
+Before upgrading, we need to prepare your postgres instance.
+This is only needed if your Postgres Instance is not externalized.
+
+# Collect your postgres user's password and the PVC attached to your postgres instance
+export POSTGRESQL_PASSWORD=$(kubectl get secret --namespace <namespace> postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode)
 export POSTGRESQL_PVC=$(kubectl get pvc --namespace <namespace> -l app.kubernetes.io/instance=circleci-server,role=primary -o jsonpath="{.items[0].metadata.name}")
+# remove the postgres statefulset without terminating your postgres instance
 kubectl delete statefulsets.apps postgresql --namespace <namespace> --cascade=orphan
+# remove the existing secret containing your postgres password. This will get recreated during upgrade.
 kubectl delete secret postgresql --namespace <namespace>
 -------------------------------------------------------------------------
+## Helm Diff
+The Helm Diff tool is used to verify that the changes between your current install and the upgrade are expected
 
-Helm Diff command (only changes) -
+# diff command
+helm diff upgrade <release-name> -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml --show-secrets --context 5 oci://cciserver.azurecr.io/circleci-server --version 4.0.0
 -------------------------------------------------------------------------
-helm diff upgrade <release-name> -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml --show-secrets --context 5 <chart-directory>
+## Helm Upgrade CircleCI Server
 
-Helm upgrade command -
--------------------------------------------------------------------------
-cd <chart-directory>
-helm dep update
-helm upgrade <release-name> -n <namespace> -f <path>/kots-exporter/output/helm-values.yaml <chart-directory> --force
+
+
```

Comment:
```suggestion
[source,shell]
```

Reply:
this is also part of the output of the migration command